testForm.js

```diff
@@ -0,0 +1,18 @@
+const forms = require('forms');
```

Comment:
This test file needs to be removed from the commit as it has nothing to do with any of the forms we are currently using.
Any future tests that you might have for your routes put them under a "test" directory.